release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -203,6 +207,8 @@ For more information about scaling a user-provisioned cluster by using the BMO,
 ==== Specifying multiple failure domains for your cluster on VSphere
 As an administrator, you can specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance. This means that you can distribute key control planes and workload elements among varied hardware resources for a datacenter. Additionally, you can configure your cluster to use a multiple layer 2 network configuration, so that data transfer among nodes can span across multiple networks.
 
+For more information, see xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#post-install-vsphere-zones-regions-configuration_post-install-vsphere-zones-regions-configuration[Specifying multiple failure domains for your cluster on VSphere].
```

Comment:
This link takes me to "Multiple regions and zones configuration for a cluster on vSphere" not "Specifying multiple failure domains for your cluster on VSphere" Maybe the link text needs changing?

Reply:
Thanks for spotting this. I updated the link to now point to the specific module.